docs/link.md

```diff
@@ -14,14 +14,20 @@ USAGE
   $ symbol-bootstrap link
 
 OPTIONS
-  -h, --help           It shows the help of this command.
-  -t, --target=target  [default: target] The target folder where the symbol-bootstrap network is generated
-  -u, --url=url        [default: http://localhost:3000] the network url
-  --maxFee=maxFee      [default: 100000] the max fee used when announcing (absolute)
-  --unlink             Perform "Unlink" transactions unlinking the voting and VRF keys from the node signer account
+  -h, --help              It shows the help of this command.
+  -t, --target=target     [default: target] The target folder where the symbol-bootstrap network is generated
+  -u, --url=url           [default: http://localhost:3000] the network url
+
+  --maxFee=maxFee         the max fee used when announcing (absolute). The node min multiplier will be used if it is not
+                          provided.
+
+  --unlink                Perform "Unlink" transactions unlinking the voting and VRF keys from the node signer account
+
+  --useKnownRestGateways  Use the best NGL known node when announcing. Otherwise the command will use the --url provided
+                          host.
```

Comment:
```suggestion
  --useKnownRestGateways  Use the best NEM node available when announcing. Otherwise the command will use the node provided
                          by the --url parameter.
```